hibernate-core/src/main/java/org/hibernate/boot/registry/selector/spi/StrategySelector.java

```diff
@@ -19,9 +19,9 @@
  * StrategySelector manages resolution of particular implementation by (possibly short) name via the
```

Comment:
StrategySelector was reverted back to its 5.1 version (without the new methods).